Maestro/subscriptions.json

```diff
@@ -1850,6 +1850,25 @@
         }
       }
     },
+    // Automate opening PRs to merge services' production branches into master
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/arcade-services/blob/production/**/*",
+        "https://github.com/dotnet/arcade-pool-provider/blob/production/**/*",
+        "https://github.com/dotnet/helix-service/blob/production/**/*",
```

Comment:
I don't recognize this one. Is it supposed to be https://dev.azure.com/dnceng/internal/_git/dotnet-helix-service?

Reply:
This is the inter-branch merges, not mirroring.

Reply:
Yeah, just wondering then, since that github repo doesn't exist

Reply:
Synced up. The repo exists, I just don't have permissions to see it. 😶